### PR TITLE
Hub Fixes

### PR DESCRIPTION
**Pull request by mrinehart-r7**
### Description

Describe the proposed changes:

  - Fix unnecessary backslashes
  - Fix various typos

### Testing

`gsed -i 's^\\:^:^g' */help.md`
```
mrinehart@BOS-MBP-4466:~icon-plugins|hub_fix_backslashes
⇒  grep '\\:' */help.md | awk -F / '{print $1}' | uniq | wc -l
       0
```

## PR Requirements

Developers, verify you have completed the following items by checking them off:

### Style

Review the [style guide](https://komand.github.io/python/style.html)

- [ ] For dependencies, pin [OS package](https://komand.github.io/python/style.html#dockerfile) and [Python package](https://komand.github.io/python/style.html#requirements-txt) versions
- [ ] For security, set least privileged account with ``USER nobody`` in the ``Dockerfile`` when possible
- [ ] For size, use the [slim SDK images](https://komand.github.io/python/sdk.html#sdk-versions) when possible: ``komand/python-3-37-slim-plugin`` and ``komand/python-3-37-plugin``
- [ ] For error handling, use of [PluginException](https://komand.github.io/python/error_handling.html#plugin-exceptions) and [ConnectionTestException](https://komand.github.io/python/error_handling.html#connection-exceptions)
- [ ] For logging, use [self.logger](https://komand.github.io/python/sdk.html#logging)
- [ ] For docs, use [changelog style](https://komand.github.io/python/style.html#changelog)
- [ ] For docs, validate markdown with ``make validate`` which calls ``mdl`` to lint ``help.md``

### Functional Checklist
- [ ] Work fully completed
- [ ] Functional
  - [ ] Any new actions/triggers include JSON [test files](https://komand.github.io/python/style.html#tests) in the `tests/` directory created with `icon-plugin run -c sample $action > tests/$action.json`
  - [ ] Tests should all pass unless it's a negative test. Negative tests have a naming convention of `tests/$action_bad.json`
  - [ ] Unsuccessful tests should fail by raising an exception causing the plugin to die and an object should be returned on successful test
  - [ ] Add functioning test results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -T tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -T all --debug --jq` (use PR format at end)
  - [ ] Add functioning run results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -R tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -R all --debug --jq` (use PR format at end)

### Assessment

You must validate your work to reviewers:

1. Run `make validate` and make sure everything passes
2. Run the assessment tool: `icon-plugin run -A -R all -T all`. For single action validation: `icon-plugin run -A -R tests/my_action.json -T tests/my_action.json`
3. Copy (`icon-plugin ... | pbcopy`) and paste the output in **a new post** on this PR.
4. Add UI screenshot of the workflow used for testing
5. Add UI screenshot of the job output used for testing
6. Add UI screenshot of the artifact (See rules in UI Checklist) used for testing

### CODE REVIEW

**Comment by jschipp-r7** [comment]
@mrinehart-r7 Is this still a WIP PR?

**Comment by mrinehart-r7** [comment]
> @mrinehart-r7 Is this still a WIP PR?

It can be merged

**Comment by jschipp-r7** [comment]
Waiting for changes to be correct and merged from https://github.com/rapid7/insightconnect-plugins/pull/217 - which has some duplication. Will take care of this after that.

File: bhr/help.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ The connection configuration accepts the following parameters:
 |Name|Type|Default|Required|Description|Enum|
 |----|----|-------|--------|-----------|----|
 |ssl_no_verify|boolean|None|True|SSL No Verify|None|
-|token|credential_token|None|True|API Token and URL to BHR Host E.g. http\://bhr.company.com\:8000|None|
+|token|credential_token|None|True|API Token and URL to BHR Host E.g. http://bhr.company.com:8000|None|
```
Comment (jschipp-r7) [inline bhr/help.md]:
```
Validator RegenerationValidator failed!
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/icon_validator/validate.py", line 29, in validate
    v.validate(spec)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/icon_validator/rules/regeneration_validator.py", line 279, in validate
    handler.run_from_validator()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/icon_validator/rules/regeneration_validator.py", line 143, in run_from_validator
    raise Exception("Error: Hashes between provided plugin and checksum were not equal! "
Exception: Error: Hashes between provided plugin and checksum were not equal! Regenerate the plugin and push to working branch.
[*] Executing validator HelpInputOutputValidator
Output violations: Action/Trigger -> "Block Address": Missing ['|result|object|False|Block Information|'] in help.md
Output violations: Action/Trigger -> "Unblock Address": Missing ['|result|string|False|Unblock Information|'] in help.md
Output violations: Action/Trigger -> "Query Block History": Missing ['|result|[]object|False|Block List|', '|is_blocked|boolean|False|Blocked|'] in help.md
Output violations: Action/Trigger -> "Block Stats": Missing ['|current|integer|False|Current blocks|', '|expected|integer|False|Expected blocks|', '|block_pending|integer|False|Blocks pending|', '|unblock_pending|integer|False|Unblocks pending|'] in help.md
Output violations: Action/Trigger -> "List Blocks": Missing ['|list|string|False|Block list|'] in help.md
Validator HelpInputOutputValidator failed!
```

File: cif/help.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ The connection configuration accepts the following parameters:
 
 |Name|Type|Default|Required|Description|Enum|
 |----|----|-------|--------|-----------|----|
-|url|string|None|True|CIF URL e.g. https\://cif.example.com|None|
+|url|string|None|True|CIF URL e.g. https://cif.example.com|None|
```
Comment (jschipp-r7) [inline cif/help.md]:
```
Validator JSONValidator failed!
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/icon_validator/validate.py", line 29, in validate
    v.validate(spec)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/icon_validator/rules/json_validator.py", line 20, in validate
    raise Exception(f"The following test files are not in proper JSON format: {JSONValidator.invalid_files}")
Exception: The following test files are not in proper JSON format: ['query1.json']
[*] Executing validator OutputValidator
[*] Executing validator RegenerationValidator
Validator RegenerationValidator failed!
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/icon_validator/validate.py", line 29, in validate
    v.validate(spec)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/icon_validator/rules/regeneration_validator.py", line 279, in validate
    handler.run_from_validator()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/icon_validator/rules/regeneration_validator.py", line 143, in run_from_validator
    raise Exception("Error: Hashes between provided plugin and checksum were not equal! "
Exception: Error: Hashes between provided plugin and checksum were not equal! Regenerate the plugin and push to working branch.
[*] Executing validator HelpInputOutputValidator
Output violations: Action/Trigger -> "Query": Missing ['|query|[]result|False|Query Results|'] in help.md
Output violations: Action/Trigger -> "Ping": Missing ['|timestamp|[]integer|True|Timestamp|'] in help.md
Validator HelpInputOutputValidator failed!
```

File: cisco_firepower/help.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ The connection configuration accepts the following parameters:
 |Name|Type|Default|Required|Description|Enum|
 |----|----|-------|--------|-----------|----|
 |certificate_passphrase|credential_secret_key|None|False|The passphrase to access the certificate|None|
-|port|integer|8307|False|Enter the port to connect to the Host Input API (Default\:8307)|None|
+|port|integer|8307|False|Enter the port to connect to the Host Input API (Default:8307)|None|
```
Comment (jschipp-r7) [inline cisco_firepower/help.md]:
```
Validator RegenerationValidator failed!
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/icon_validator/validate.py", line 29, in validate
    v.validate(spec)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/icon_validator/rules/regeneration_validator.py", line 279, in validate
    handler.run_from_validator()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/icon_validator/rules/regeneration_validator.py", line 143, in run_from_validator
    raise Exception("Error: Hashes between provided plugin and checksum were not equal! "
Exception: Error: Hashes between provided plugin and checksum were not equal! Regenerate the plugin and push to working branch.
[*] Executing validator HelpInputOutputValidator
Output violations: Action/Trigger -> "List All Incidents": Missing ['|incidents|[]incident|False|Incidents|'] in help.md
Output violations: Action/Trigger -> "List All Organization Policies": Missing ['|policies|[]policy|False|Policies|'] in help.md
Output violations: Action/Trigger -> "List All Organization Applications": Missing ['|applications|[]application|False|Applications|'] in help.md
Output violations: Action/Trigger -> "List Activities": Missing ['|activities|[]activity|False|Activities|'] in help.md
Output violations: Action/Trigger -> "List Entities": Missing ['|entities|[]entity|False|Entities|'] in help.md
Output violations: Action/Trigger -> "List All Suspicious IP Entries": Missing ['|entries|[]suspicious_ip_entry|False|Suspicious IP entries|'] in help.md
```

**Pull request merged**